fp.js

```diff
@@ -24,11 +26,23 @@ people.forEach(person => {
   }
 })
 
+function isYoungerThan(age) {
+  return people.age <35;
+  var peopleYoungerThan35 = people.filter(isYoungerThan);
+}
```

Comment:
close, careful about which variable you are passing and referencing in the `isYoungerThan` function, also make sure to call the filter outside the `isYoungerThan` function, i.e:
```js
function isYoungerThan (person) {
 return person.age < 35
}
var peopleYoungerThan35 = people.filter(isYoungerThan)
```